orchestrator/relayer/src/main_loop.rs

```diff
@@ -88,8 +200,17 @@ pub async fn relayer_main_loop(
             )
             .await;
         }
-
-        if should_relay_batches {
+        let delay_altruistic_relayer = relayer_config.batch_relaying_mode
+            == BatchRelayingMode::Altruistic
+            && Instant::now() - *RELAYER_START
+                < Duration::from_secs(relayer_config.altruistic_batch_relaying_delay);
+        if delay_altruistic_relayer {
+            info!(
+                "Delaying relayer because {} seconds have not elapsed",
```

Comment:
I'm not seeing an actual delay here?

It may be slightly nicer to use a sample count function off of the gas tracker and simply have the relayer wait until there are 5 samples, cleaner than time based delays in my opinion.